lib/themes/utils/transformForTokens.js

```diff
@@ -0,0 +1,31 @@
+import merge from 'lodash/merge';
+import alignTextToGrid from './alignTextToGrid';
+import createResponsiveRules from './createResponsiveRules';
+
+export default ({ tokens }) => {
+  const rules = [];
+  Object.keys(tokens.text).forEach(typeName => {
```

Comment:
Just a personal preference here but could we maybe reduce this into the object? That way we don't need merge.

Reply:
I prefer reducing, but to clean it up would mean following this down into `createResponsiveRules` and everywhere that is used. I would like to come back and convert all the `utils` to TypeScript once the components are done.